svg/elements/stop.json

```diff
@@ -90,6 +91,8 @@
         },
         "stop-color": {
           "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/stop-color",
```

Comment:
Rather than adding this MDN url manually, maybe we should add this BCD key to the content page, and then the content-inventory-related automation will automatically set this url.

Reply:
Never mind, this seems to be what we're also doing with `stop-opacity`. And this makes a useful link from https://developer.mozilla.org/en-US/docs/Web/SVG/Element/stop#browser_compatibility to the `stop-color` page. 👍